doc/classes/Control.xml

```diff
@@ -21,13 +21,16 @@
 			<return type="bool">
 			</return>
 			<description>
+				Virtual method to be implemented by the user. Returns whether [method _gui_input] should not be called for children controls outside this control's rectangle. Input will be clipped to the Rect of this [code]Control[/code]. Similar to [member rect_clip_content], but doesn't affect visibility.
```

Comment:
When referring to the current class, you can still use `[Control]` instead of `[code]Control[/code]`. We initially advised for the latter to avoid having too many useless hyperlinks, but it's handled directly in the HTML generation now.